src/asciidoc/reference/packages/withkojiCustomVccSdk.adoc

```diff
@@ -27,45 +24,52 @@ Instantiates `CustomVcc`.
 
 [source,javascript]
 ----
-import { CustomVcc } from '@withkoji/custom-vcc-sdk';
+import CustomVcc from '@withkoji/custom-vcc-sdk';
 const customVcc = new CustomVcc();
 ----
 
-[.hcode, id=".register", reftext="register"]
-=== .register(width, height)
+== CustomVcc methods
 
-Indicates that the custom VCC has loaded and registers it to trigger the `on` events from the parent editor.
+[.hcode, id=".change", reftext="change"]
+=== .change(newValue)
+
+Updates the VCC value with the user-entered value.
 
 ==== Parameters
 
-* `width` – Any, width of the custom VCC.
-* `height` – Any, height of the custom VCC.
+* `newValue` – _Any_, current value of the VCC.
 
 ==== Example
 
 [source,javascript]
 ----
-customVcc.register('300', '300');
+customVcc.change(newValue);
 ----
 
-[.hcode, id=".change", reftext="change"]
-=== .change(newValue)
+[.hcode, id=".onTheme", reftext="onTheme"]
+=== .onTheme(handler)
 
-Updates the VCC value with the user-entered value.
+Listens to changes to the theme value, to retrieve values relating to the Koji display theme.
+The theme returned in the listener allows you to style the VCC to match the user’s current theme on Koji (dark or light).
 
 ==== Parameters
 
-* `newValue` – Any, current value of the VCC.
+* `handler` – _Function_, called when theme is available.
+Receives the following property as input:
+** `theme` – _Object_, represents the theme as an object with CSS colors and mixins.
```

Comment:
Similar to the `props` object in `onUpdate`, it would be very helpful to have more info about the structure. Again, we could follow a similar pattern and pull out the info into a section that's linked from here. However, since this object is huge and it wouldn't be helpful to explain every property in it, we can provide a complete example and then just call out a few particularly useful properties in it. That should be enough to demonstrate the pattern for developers.